src/Styles/Hooks.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace WPML\PB\Cornerstone\Styles;
+
+use WPML\FP\Fns;
+use WPML\FP\Maybe;
+use WPML\LIB\WP\Post;
+use function WPML\FP\partialRight;
+
+class Hooks implements \IWPML_Backend_Action, \IWPML_Frontend_Action, \IWPML_DIC_Action {
+
+	const META_KEY = '_cs_generated_styles';
+
+	/** @var \WPML_PB_Last_Translation_Edit_Mode $lastEditMode */
+	private $lastEditMode;
+
+	/** @var \WPML_Cornerstone_Data_Settings $dataSettings */
+	private $dataSettings;
+
+	/**
+	 * Hooks constructor.
+	 *
+	 * @param \WPML_PB_Last_Translation_Edit_Mode $lastEditMode
+	 * @param \WPML_Cornerstone_Data_Settings     $dataSettings
+	 */
+	public function __construct(
+		\WPML_PB_Last_Translation_Edit_Mode $lastEditMode,
+		\WPML_Cornerstone_Data_Settings $dataSettings
+	) {
+		$this->lastEditMode   = $lastEditMode;
+		$this->dataSettings   = $dataSettings;
+	}
+
+
+	public function add_hooks() {
+		add_action( 'save_post', [ $this, 'invalidateStylesInTranslation' ] );
+	}
+
+	public function invalidateStylesInTranslation( $postId ) {
+		Maybe::of( $postId )
+			->filter( [ $this->dataSettings, 'is_handling_post' ] )
+			->filter( [ $this->lastEditMode, 'is_translation_editor' ] )
+			->map( Post::deleteMeta( Fns::__, self::META_KEY, '' ) );
```

Comment:
@strategio `Post::deleteMeta` only takes two arguments ( in a way ). You don't need the third `''`

Reply:
Of course Bruce!